README.md

```diff
@@ -1,3 +1,85 @@
-# meta-astertom
+# ![Astertom](images/Astertom-logo-300-71.jpg)
 
-Base yocto layer for Astertom distribution
+## meta-astertom
+
+Yocto layer for building Astertom distribution for reference boards.
+
+## Supported boards
+
+- raspberrypi4-64
+
+## Dependencies
+
+This layer depends on additional layers:
+
+- [poky](http://git.yoctoproject.org/cgit/cgit.cgi/poky)
+- [meta-openembedded](http://git.openembedded.org/meta-openembedded)
+
+Optional:
+
+- [meta-raspberrypi](http://git.yoctoproject.org/cgit.cgi/meta-raspberrypi)
+
+## Environment preparation
+
+### General informations
+
+Before starting building, make sure that all required tools are installed on your host machine. For more informations how to configure your machine visit [Yocto project page](https://yoctoproject.org)
+
+### By ansible playbook
+
+You can use [setup_layser.yml](./setup_layers.yml) [ansible](https://www.ansible.com/) playbook to clone all required layers inside specific directory on your host machine, and set example configuration files: `local.conf` and `bblayers.conf`
+
+To setup environmet by ansible playbook and clone all required layers into specific directory run:
```

Comment:
Is "and" necessary here? It looks like "ansible playbook" and "clone all" are separate tools. But when you are reading the whole sentence it seems that "and" should be deleted.